### PR TITLE
Porting for First Playtest

### PR DESCRIPTION
**Pull request by gusxyz**
# Description.
Just porting a lot of stuff from foundation 19.
This will give us a good basis for the first play test..

## General
- [x] ID cards
- [x] Departmental Utility Coveralls
- [x] Departmental Jumpsuits

## Class-D
- [x] Class-D Jumpsuit

## Security 
- [x] LCZ Guard Jumpsuit
- [x] HCZ Guard Jumpsuit
- [x] Guard Medium Armor
- [X] Guard Commander
- [X] Guard Helmet

## Research
- [x] Labcoat
- [x] Suits
- [X] Research Director Turtle Neck

## Logistics
- [x] Jumpsuits
- [x] Coveralls

## Maintenace
- [X] Maintenace Director Jumpsuit
- [x] Jumpsuits
- [x] Coveralls

## Medical
- [X] Medical Director Turtle Neck
- [X] Suits
- [X] Labcoats

# Media
Content Client_nXL8T0SuoZ
Content Client_JnpJgCEJ8Z

# Changelog

:cl:
- add: Added a lot of basic entities related to characters.

### CODE REVIEW

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+﻿- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: SCPClothingOuterCoatLabSci
+  name: foundation researcher's lab coat
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml:4]: change to just researcher's

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    sprite: _SCP/Clothing/OuterClothing/Coats/scilabcoat.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.75
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, SCPClothingOuterCoatLabSci]
+  id: SCPClothingOuterCoatLabSciOpened
+  name: foundation researcher's lab coat
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml:19]: change to just researcher's

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: SCPClothingOuterCoatLabAsst
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml:23]: remove reduntant clothing

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  description: A coat that protects against minor chemical spills. It has a SCP logo on the chest, it doesn't have much to show in designation.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/OuterClothing/Coats/asstlabcoat.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/OuterClothing/Coats/asstlabcoat.rsi
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, SCPClothingOuterCoatLabAsst]
+  id: SCPClothingOuterCoatLabAsstOpened
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml:34]: remove reduntant clothing

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, SCPClothingOuterCoatLabRD]
+  id: SCPClothingOuterCoatLabRDOpened
+  name: research director's lab coat
+
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: SCPClothingOuterCoatLabMD
+  name: medical director's lab coat
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml:56]: director of X naming convention

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/OuterClothing/Coats/mdlabcoat.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/OuterClothing/Coats/mdlabcoat.rsi
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, SCPClothingOuterCoatLabMD]
+  id: SCPClothingOuterCoatLabMDOpened
+  name: medical director's lab coat
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml:67]: director of X naming convention

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/vests.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
+- type: entity
+  parent: ClothingOuterArmorBasic
+  id: ClothingOuterSCPGuardMedArmor
+  name: foundation security vest
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/vests.yml:4]: just security

File: Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/lczguard.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/lczguard.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformSecurityLCZSeargent
+  name: lcz seargent uniform
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/jumpsuits.yml:63]: remove all LCZ or HCZ sec specific clothing

File: Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/engineeringdirector.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/engineeringdirector.rsi
+
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformUtility
+  name: utility coveralls
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/jumpsuits.yml:108]: remove

File: Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/suits.yml [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Suits/senresearcher.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Suits/senresearcher.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformMedicalDirectorTurtleneck
+  name: medical director's turtleneck
```
Comment (JCGWE30) [inline Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/suits.yml:27]: Director of X naming conventions

**Review by JCGWE30 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/jumpsuits.yml (hunk 9 above)
Comment (gusxyz) [reply Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/jumpsuits.yml:108]: It's the parent and the base for the department uniforms, it also works as an unmarked coveralls for other personnel.

File: Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml (hunk 3 above)
Comment (gusxyz) [reply Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml:23]: This was meant to be different than the RD coat... I had just broken the rd coat rsi file previously and never readded it.

**Comment by Napkino** [comment]
Main issue with this right now is that the RSIs didn't validate. once that issue is fixed and the workflows pass, this is clear to merge.

**Pull request merged**